coderd/database/queries/insights.sql

```diff
@@ -357,7 +357,8 @@ WITH
 		SELECT
 			COALESCE(
 				MAX(start_time) - '1 hour'::interval,
-				NOW() - '6 months'::interval
+				-- TODO(mafredri): Fix this, required for tests to pass.
```

Comment:
Follow-up PR?

Reply:
Fixed by changing the hard-coded nature of this query to use actual data from `workspace_agent_stats`.